apollo-runtime/src/main/java/com/apollographql/android/cache/normalized/lru/EvictionPolicy.java

```diff
@@ -0,0 +1,117 @@
+package com.apollographql.android.cache.normalized.lru;
+
+import android.app.ActivityManager;
+import android.content.Context;
```

Comment:
Only one concern if we want to keep runtime pure java, we shouldn't have android deps